src/AlpineSkiHouse.Web/Models/Scan.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace AlpineSkiHouse.Models
+{
+    public class Scan
+    {
+        public int Id { get; set; }
+
+        public int CardId { get; set; }
```

Comment:
Hmm...well there is only a `CardId` on here. 

So to think through that...a card being scanned would not be valid given any current activations, thus the business logic would loop back to see if there was a valid, not-yet-activated pass. Okay...that seems to make sense.